constants/tasks.js

```diff
@@ -5,11 +5,34 @@ const TASK_TYPE = {
 };
 
 const TASK_STATUS = {
-  ACTIVE: "active",
-  ASSIGNED: "assigned",
-  BLOCKED: "blocked",
-  COMPLETED: "completed",
-  UNASSIGNED: "unAssigned",
+  // old task workflow
+  ACTIVE: "IN_PROGRESS",
+  "HIGH PRIORITY": "HIGH PRIORITY",
```

Comment:
You can remove quotes from the key.
https://github.com/airbnb/javascript#objects--quoted-props

Reply:
Okay. Got it.